### ISSUE TITLE
Inline suggestion hover should support better keyboard navigation

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#144060]
It would be great if arrow keys could be used for keyboard navigation.
That way I do not have to have "tab key moves focus" to use tab to move focus inside the inline suggestion hover.

**Comment by hediet** [comment]
I think this should be a general feature request for all hover messages.

**Comment by isidorn** [comment]
Yeah makes sense. Only special about the inline suggestion hover is that it has that part with multiple actions. Though I am sure other hovers have as well.

So I guess hovers would need some common infra to tell if they have navigable "buttons".

**Comment by kimadeline** [comment]
Hello, Copilot dev here 🧃 

We've had reports of lack of screen reader feedback regarding inline suggestions as well, specifically for the following scenarios:
- presence of a suggestion
- accepting and dismissing a suggestion
- using actions displayed in the inline suggestions menu: cycling through suggestions + whichever custom actions we add (in this case, the "Open GitHub Copilot" action)

Let me know if there's anything we could do to help, whether it is collecting feedback, brainstorming ideas, or anything else!

**Comment by isidorn** [comment]
@kimadeline thanks for sharing that feedback. I think we can start with brainstorming ideas. Do you have any suggestions on how to improve the current flow?

**Comment by benjiallen** [comment]
@isidorn thinking through a response to [your comment above](https://github.com/microsoft/vscode/issues/144060#issuecomment-1398176370).

Can you provide a screenshot to help me understand what you mean by "inline suggestion hover"? I'm trying to understand how much overlap there is with the problem documented in this issue vs. other issues mentioned by @kimadeline

**Comment by benjiallen** [comment]
![Screenshot of VS Code with Copilot enabled. Copilot has a suggestion and upon hovering over the suggestion a list of options is shown.](https://user-images.githubusercontent.com/16546334/215181102-05666ac3-7e86-43aa-8a7d-82a7650c8843.png)

@isidorn is this an example of the "inline suggestion hover"?

**Comment by isidorn** [comment]
Yes, but we have redesigned it in VS Code Insiders. It should hit stable in a week.

**Comment by gdomingu** [comment]
Hi there, another copilot dev here. Looking through open accessibility issues and found a link to this one. Wondering if there have been any updates on this? It seems like the `^Enter` shortcut does open the copilot panel, but I don't see the text in the hover menu anymore. Does this https://github.com/microsoft/vscode/issues/144060#issuecomment-1406989338 mean it can be closed now?

**Comment by isidorn** [comment]
Yes. I think we can close this now, and we can file new issue for further improvements of the Inline Suggestions accessibility. Thanks
